scripts/build.sh

```diff
@@ -17,7 +17,7 @@ BIN=$(basename $(go list -f '{{.ImportPath}}'))
 echo "Building $BIN $VERSION"
 ## Adding GOTAGS so that mgmt builds without the libvirt and augeas resrouces on centos-ci
 ## TODO Remove this once mgmt can be built engine only without resources
-GOTAGS='novirt noaugeas' go build -ldflags "${LDFLAGS}" -o $OUTDIR/$BIN || exit 1
+go build -ldflags "${LDFLAGS}" -o $OUTDIR/$BIN -tags "novirt noaugeas" || exit 1
```

Comment:
you might want to use build -i https://ttboj.wordpress.com/2017/02/26/faster-golang-builds/